client-base/environment.yml

```diff
@@ -11,4 +11,5 @@ dependencies:
   - python-dotenv
   - python-multipart
   - rich
-  - typer
+  # fixed version because of https://github.com/DIRACGrid/diracx/issues/280
+  - typer=0.12.3
```

Comment:
```suggestion
  - typer==0.12.3
```
?

Reply:
Apparently this single = is correct here? (In requirements.txt / pip one needs two)

Reply:
I changed to exclude version 0.12.4

Reply:
Yes but if they release `0.12.5` with the same issue ?

Reply:
Then we'll think about it... or do

```
typer >0.12.5,<0.12.4
```